tests/integration/cli/test_run.py

```diff
@@ -1,36 +1,27 @@
-from .utils import skip_projects, skip_projects_except
+from .utils import skip_projects_except
 
 BAD_COMMAND = "not-a-name"
 
 
-@skip_projects(["script", "geth"])
-def test_run_no_scripts_dir(ape_cli, runner, project):
-    result = runner.invoke(ape_cli, ["run", BAD_COMMAND])
-    assert result.exit_code == 1, result.output
-    assert "No 'scripts/' directory detected to run script" in result.output
-
-
-@skip_projects_except(["script"])
-def test_run_no_argument(ape_cli, runner, project):
-    result = runner.invoke(ape_cli, ["run"])
-    assert result.exit_code == 1, result.output
-    assert "Must provide at least one script name or path" in result.output
-
-
 @skip_projects_except(["script"])
 def test_run_unknown_script(ape_cli, runner, project):
     result = runner.invoke(ape_cli, ["run", BAD_COMMAND])
-    assert result.exit_code == 1
-    assert f"No script named '{BAD_COMMAND}' detected in scripts folder" in result.output
+    assert result.exit_code == 2
+    assert f"No such command '{BAD_COMMAND}'." in result.output
 
 
-@skip_projects(
-    ["empty-config", "no-config", "one-interface", "unregistered-contracts", "test", "geth"]
-)
 def test_run(ape_cli, runner, project):
-    for script_file in project.scripts_folder.glob("*.py"):
-        if script_file.stem == "__init__":
-            continue
+    result = runner.invoke(ape_cli, ["run"])
+    assert result.exit_code == 0, result.output
```

Comment:
what is the thought behind allowing ape run to work without any arguments?

I would think a script name should be required, no?

Reply:
To display help information about `ape run`, as well as display all the scripts that you can execute from the `scripts/` folder
```bash
$ ape run
  Run scripts from the ``scripts`` folder. ...

Options:
  -i, --interactive  Drop into interactive console session after running
  -h, --help         Show this message and exit.

Commands:
  click   Use a subcommand
  deploy  Run 'scripts/deploy.py:main'
  site    Run 'scripts/site.py'
```

Reply:
That is neat!